tests/scripts/test-ci.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+set -ex
```

Comment:
```suggestion
set -xeuo pipefail
```

`set -euo pipefail` is the standard set of script options that most bash scripts start with. I like that you have `x` here for debugging also.

`pipefail` will make the script fail if a pipe step fails. E.g., if B fails here `A | B | C`, it counts as a script failure.

`u` will make sure any variables that are undefined also cause a failure, which is important to prevent things like `docker tag "$image" quay.io/tag:blah` from interpreting as `docker tag "" quay.io/tag:blah`